modules/aws/auth.go

```diff
@@ -153,3 +153,48 @@ type CredentialsError struct {
 func (err CredentialsError) Error() string {
 	return fmt.Sprintf("Error finding AWS credentials. Did you set the AWS_ACCESS_KEY_ID and AWS_SECRET_ACCESS_KEY environment variables or configure an AWS profile? Underlying error: %v", err.UnderlyingErr)
 }
+
+func getNewSession(c *aws.Config) (*session.Session, error) {
+	var config *aws.Config
+	customConfig := GetCustomConfig()
+
+	if customConfig != nil {
+		config = getAwsConfigFromCustomConfig(customConfig)
```

Comment:
This is ignoring the passed in config (`c`)... So tests that set a specific region or passed in custom credentials (see the usage of `getNewSession` above) won't end up actually using them, which is going to be quite confusing. I think you'll need to merge the passed-in config and the custom config.